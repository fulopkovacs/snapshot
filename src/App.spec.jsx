import {expect, it} from 'vitest'
import renderer from 'react-test-renderer'
import App from './App'

it('renders <App> properly', () => {
  const tree = renderer.create(<App />)
  expect(tree).toMatchSnapshot()
})
