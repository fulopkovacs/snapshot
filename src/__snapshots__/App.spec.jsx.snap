// Vitest Snapshot v1

exports[`renders <App> properly 1`] = `
<div
  className="App"
>
  <button
    onClick={[Function]}
  >
    count is 
    0
  </button>
</div>
`;
